### PR TITLE
add TxGenerator

### PR DESCRIPTION
**Pull request by cyjseagull**

### CODE REVIEW

File: libprecompiled/Common.h [hunk 1]
```diff
@@ -21,6 +21,7 @@
 #pragma once
 #include "libdevcore/Exceptions.h"
 #include "libprecompiled/Precompiled.h"
+#include <libdevcore/Address.h>
```
Comment (bxq2011hust) [inline libprecompiled/Common.h:24]: seems unnecessary?

File: libprecompiled/Precompiled.h [hunk 2]
```diff
@@ -20,6 +20,7 @@
  */
 #pragma once
 
+#include "Common.h"
```
Comment (bxq2011hust) [inline libprecompiled/Precompiled.h:23]: only precompiled.cpp need this?

Comment (bxq2011hust) [reply libprecompiled/Precompiled.h:23]: because many Precompiled need getParamFunc?

Comment (cyjseagull) [reply libprecompiled/Precompiled.h:23]: Yes, many Precompiled need `getParamFunc`, so I put `Common.h` into `Precompiled.h`.

File: libblockverifier/ExecutiveContextFactory.cpp [hunk 3]
```diff
@@ -55,34 +55,34 @@ void ExecutiveContextFactory::initExecutiveContext(
...
     {
-        context->setAddress2Precompiled(
-            Address(0x1007), std::make_shared<dev::precompiled::ContractLifeCyclePrecompiled>());
+        context->setAddress2Precompiled(ADDRESS_CONTRACT_LIFECYCLE,
+            std::make_shared<dev::precompiled::ContractLifeCyclePrecompiled>());
         auto kvTableFactoryPrecompiled =
             std::make_shared<dev::precompiled::KVTableFactoryPrecompiled>();
         kvTableFactoryPrecompiled->setMemoryTableFactory(memoryTableFactory);
-        context->setAddress2Precompiled(Address(0x1010), kvTableFactoryPrecompiled);
+        context->setAddress2Precompiled(ADDRESS_KVTABLE_FACTORY, kvTableFactoryPrecompiled);
```
Comment (bxq2011hust) [inline libblockverifier/ExecutiveContextFactory.cpp:80]:
```suggestion
        context->setAddress2Precompiled(KVTABLE_FACTORY_ADDRESS, kvTableFactoryPrecompiled);
```

Comment (cyjseagull) [reply libblockverifier/ExecutiveContextFactory.cpp]: Done.

**Pull request merged**